frontends/open-discussions/src/components/MetaTags.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
```

Comment:
No changes here besides removing TS definitions. This is generally the same as ol-util/src/components/MetaTags.ts.